storagenode/orders/sender.go

```diff
@@ -105,16 +106,18 @@ func (sender *Sender) Run(ctx context.Context) error {
 
 		if len(ordersBySatellite) > 0 {
 			var group errgroup.Group
-			ctx, cancel := context.WithTimeout(ctx, sender.config.Timeout)
-			defer cancel()
 
```

Comment:
Why was this moved inside the Go func?

Reply:
To have timeout per satellite. The same way how you did it in `agreementsender`.

Reply:
There's no need to create multiple contexts, because the timeout will be shared between them.

Reply:
Earlier sounds like a good idea :) I will change this.